frontend/components/Map.js

```diff
@@ -15,11 +15,14 @@ const Map = ({}) => {
         cooperativeGestures: true,
         container: "map",
         style: "mapbox://styles/lknarf/cl0pf3asg000114nt5jepzs6s",
-        center: [-135.32, 57.0531],
         minZoom: 8,
-        zoom: 10,
+        pitch: 60,
+        bearing: 0,
         maxZoom: 18,
-        hash: true,
+        bounds: [
+          [-135.40821075439453, 57.031763208858415],
+          [-135.19054412841797, 57.12282315984669],
```

Comment:
```suggestion
          [-135.408, 57.032],
          [-135.191, 57.123],
```
Doesn't actually matter, but for aesthetics and because https://xkcd.com/2170/